xml/System.Composition.Convention/ParameterImportConventionBuilder.xml

```diff
@@ -12,8 +12,8 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Contains methods used when configuring a <see cref="PartConventionBuilder{T}"/>. This class is used only in expressions, instances of this type are never created.</summary>
```

Comment:
Move "This class is used..." to the Remarks section. Also, a semicolon, not a comma, should be used.